docs/tutorials/executor_forces_tutorial.rst

```diff
@@ -371,15 +372,79 @@ These may require additional browsing of the documentation to complete.
             time.sleep(0.1)
             task.poll()
 
-            if task.file_exists_in_workdir(statfile):
+            if task.file_exists_ins_workdir(statfile):
```

Comment:
?

Reply:
Whoops, must have been a weird find-and-replace